src/codewhisperer/activation.ts

```diff
@@ -64,6 +64,13 @@ export async function activate(context: ExtContext): Promise<void> {
         await enableDefaultConfigCloud9()
     }
 
+    //Opens the getting started page if it is not already open.
```

Comment:
Is this comment accurate? `registerCommandsWithVSCode` only registers the command but doesn't call it, the actual command would be done elsewhere.

Reply:
Yeah That part of code was deleted and I forgot to remove this comment